caravel/templates/manage_listings.html

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.html" %}
+{% block title %} Manage Listings {% endblock %}
+{% block head %}
+  {{ super() }}
+{% endblock %}
+{% block content %}
```

Comment:
Exactly — why not just allow queries by seller email? Then "manage listings page" is just a link to "?q=myemailaddress@uchicago.edu."